src/java.base/share/classes/sun/security/jca/Providers.java

```diff
@@ -100,6 +100,20 @@ private Providers() {
         "SunJCE",
     };
 
+    // Hardcoded fully qualified names of providers to use for JAR
+    // verification when RestrictedSecurity is enabled.
+    // (Similar to jarVerificationProviders array)
+    //
+    // MUST NOT be on the bootclasspath and not in signed JAR files.
+    private static final String[] jarVerificationProvidersRS = {
```

Comment:
Suggest a better name is `restrictedJarVerificationProviders` (rather than abbreviate in `RS`).

Reply:
Changed to suggestion.